addons/product/tests/test_variants.py

```diff
@@ -401,6 +401,26 @@ def test_open_product_form_with_default_uom_id_is_false(self):
         product = product_form.save()
         self.assertEqual(uom_unit, product.uom_id)
 
+    def test_create_product_with_packaging_one_variant(self):
+        """Test what happen when creating a product with attribute,
+        with only one variant and a packaging. On creating the packaging
+        should not be duplicated."""
+
+        product_template = self.env['product.template'].create({
+            'name': 'one variant template',
```

Comment:
```suggestion
            'name': "one variant template",
```
User-facing string like record names should preferably use double quotes.